CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.2.3] - 2024-08-02
+
+### 📚 Documentation
+
+- Clean up changelogs ([#17](https://github.com/joshka/tui-widgets/pull/17))
+  > - removed unnecessary footer comments
+  > - removed [unreleased] sections
+  > - removed duplicate release notes
+
+<!-- generated by git-cliff -->
```

Comment:
Still need to fix this up in the config... cc @orhun

Reply:
Ah, let's just remove it.

Reply:
yeah - thanks for that. I figured it was probably an easy fix, only tagging you on it as a kinda indication about how the tools interact. Thanks for removing this for me.